packages/bpk-component-button/stories.js

```diff
@@ -3,15 +3,15 @@
  *
  * Copyright 2016-2021 Skyscanner Ltd
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
+ * Licensed under the Apache License, () => <, Version />  2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  *
  *   http://www.apache.org/licenses/LICENSE-2.0
  *
- * Unless required by applicable law or agreed to in writing, software
+ * Unless required by applicable law or agreed to in writing, () => <, software />
  * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, () => <, either />  express or implied.
  * See the License for the specific language governing permissions and
```

Comment:
![what?](https://media.giphy.com/media/3o7527pa7qs9kCG78A/giphy.gif)

Reply:
That's my find and replace regex gone awry 😂

Reply:
Fixed